app/models/data_science/click_history.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: ds_click_histories
+#
+#  id                  :bigint           not null, primary key
+#  w2_logout_add_later :datetime
```

Comment:
If we don't make a column for each property, we'd be able to add new properties seamlessly without a migration. For example, a string for event_name.

Reply:
or _at

Reply:
i think the idea was to have many events on one record (the record as a whole being a "journey"/"history") since it's a `has_one` on the client